hpx/plugins/parcelport/tcp/receiver.hpp

```diff
@@ -107,7 +107,7 @@ namespace hpx { namespace parcelset { namespace policies { namespace tcp
                     = &receiver::handle_read_header<Handler>;
 
                 boost::asio::async_read(socket_, buffers,
-                    boost::bind(f, shared_from_this(),
+                    util::bind(f, shared_from_this(),
                         boost::asio::placeholders::error,
                         boost::asio::placeholders::bytes_transferred,
                         util::make_tuple(handler)));
```

Comment:
Now we probably don't need to pass the handler in an one-element tuple anymore. This was a workaround for some problems with boost::bind.